src/unstract/sdk/llm.py

```diff
@@ -116,20 +117,24 @@ def get_llm(
                     ][Common.ADAPTER]
                     llm_metadata = llm_config_data.get(Common.ADAPTER_METADATA)
                     llm_adapter_class: LLMAdapter = llm_adapter(llm_metadata)
-                    llm_instance: Optional[
-                        LLM
-                    ] = llm_adapter_class.get_llm_instance()
+                    llm_instance: Optional[LLM] = (
```

Comment:
@gaya3-zipstack the return type should be `LLM` only right?

Reply:
Thank you for catching. Removed